### PR TITLE
Feature: app-center-v6

### PR DESCRIPTION
**Pull request by mdiha**
User_UI_US02: App center drawer
User_UI_US04: Long apps list
User_UI_US08: Default order
Metrics_US10.1 : Drawer/log grafana

### CODE REVIEW

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+// Primary Color
+@primaryColorDefault: #578dc9;
+@primaryColor: var(--allPagesPrimaryColor, @primaryColorDefault);
+@primaryColorLight: var(--allPagesPrimaryColorLight, lighten(@primaryColorDefault, 10%));
+
+// Other base colors
+@darkGrey: var(--allPagesDarkGrey, #4d5466);
+@greyColorLighten1Default: #a8b3c5;
+@greyColorLighten1: var(--allPagesGreyColorLighten1, @greyColorLighten1Default);
+@toolbarLightIconColor: var(--allPagesToolbarLightIconColor, @greyColorLighten1Default);
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: This should be removed to use variable.less

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 1 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Done

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 2]
```diff
@@ -836,74 +840,113 @@
...
 
-.appLauncherTitle {
-  display: block;
-  text-align: center;
-  color: #555759;
-  font-size: 16px;
+    .seeAllApplicationsBtn {
+      margin-bottom: 13px!important;
+      margin-top: 13px!important;
+      border: 1px solid #1976d27a!important;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Could you check the color here ? the color code seems not ok

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 3]
```diff
@@ -836,74 +840,113 @@
...
+    border-top: 1px solid #f0f4fe!important;
+    background: white;
 
-.appLauncherTitle {
-  display: block;
-  text-align: center;
-  color: #555759;
-  font-size: 16px;
+    .seeAllApplicationsBtn {
+      margin-bottom: 13px!important;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Why do you put "important" here ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 4]
```diff
@@ -836,74 +840,113 @@
...
+      margin-top: 13px!important;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Why do you put "important" here ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 2 above)
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Why do you put "important" here ?

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 5]
```diff
@@ -836,74 +840,113 @@
...
+      border: 1px solid #1976d27a!important;
+      padding: 8px 15px;
+      border-radius: 4px!important;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Are you sure this is mandatory ? On acceptance, if I remove this, the radius is still ok

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 6]
```diff
@@ -836,74 +840,113 @@
...
+      cursor: pointer!important;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Are you sure this is mandatory ? On acceptance, if I remove this, the radius is still ok

File: app-center-webapps/src/main/webapp/skin/less/app-center.less [hunk 7]
```diff
@@ -836,74 +840,113 @@
...
+      letter-spacing: .5px;
```
Comment (rdenarie) [inline app-center-webapps/src/main/webapp/skin/less/app-center.less]: Are you sure this is mandatory ? On acceptance, it is override by vue css

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you check all !important in css ? 
They are not all necessary. Please check the one I have commented, and the one I have not commented

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 5 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Yes it is not necessary, will remove it

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 2 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Checked

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 3 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Fixed

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 4 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Fixed

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 2 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Fixed

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 6 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Fixed

File: app-center-webapps/src/main/webapp/skin/less/app-center.less (hunk 7 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/skin/less/app-center.less]: Fixed

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — DISMISSED** [review]
LGTM

**Review by rdenarie — APPROVED** [review]
LGTM

**Pull request merged**